app/pupilfirst_api.rb

```diff
@@ -13,7 +13,9 @@ def headers(_context)
       end
     end
 
-    Schema = GraphQL::Client.load_schema('/app/graphql_schema.json')
+    schema_path = "#{ENV['GITHUB_ACTION_PATH']}/app/graphql_schema.json"
```

Comment:
Do document this somewhere, else it would be hard to run locally; 
I think you should add an `example.env` file so that others can also run the action locally;